MesaModel/model/Pupil.py

```diff
@@ -561,20 +560,20 @@ def changeState(self):
                 self.model.model_state.learning_count -= 1
             return 1
 
-    def set_start_math(self):
+    # Calculate the end maths score based on the time spent by a pupil in
+    # a learning state and the pupil's starting maths score
+    def set_end_math(self):
         # Increment the learning counter
-        self.countLearning += 1
-
-        # Scale Smath before using it to calculate end math score
-        # 8550t = 7.621204857
+        self.count_learning += 1
 
-        Scaled_Smath = (2.718281828 ** self.s_math) ** (1 / 7.621204857)
-        total_learn = self.countLearning + Scaled_Smath
+        scaled_s_math = (2.718281828 ** self.s_math) ** (1 / 7.621204857)
+        total_learn = self.count_learning + scaled_s_math
         self.e_math = (7.621204857 * math.log(total_learn)) + self.ability
 
     def get_type(self):
         return self.type
 
+    # Compute the pupil's disruptive tendency
```

Comment:
Might be worth adding more detail here about what's going on, because this isn't something that's done in the NetLogo model iirc.